css/build.css

```diff
@@ -171,3 +171,8 @@ input.form-control:focus {
 		padding: 0 12.5%;
 	}
 }
+
+/* Fix component view when it is placed in the container */
```

Comment:
@tonytlwu @hcarneiro please confirm you're happy with this change

Reply:
Can't confirm if this would have any negative impact, but the code change looks legit. The rest is up to the developer and QA to verify.

Reply:
@hcarneiro Is this necessary because D&D3 now has all content laid out like inline blocks using flexbox, and therefore this block DIV needs `width: 100%` to occupy the 100% of horizontal space?

Reply:
Yes, this will force the component to occupy the entire horizontal space, by default on the new D&D system. This can be overwritten by the Appearance Settings.
I'm fine with this change!

Reply:
@hcarneiro Thanks!